README.md

```diff
@@ -1,132 +1,37 @@
 Chitter Challenge
```

Comment:
I would love to know more about your process in creating this. What did you learn, where did you get stuck, and how did you plan?